packages/client/src/components/reports/canary/Metric.tsx

```diff
@@ -12,15 +16,26 @@ export default class Metric extends React.Component<Props> {
   render(): React.ReactNode {
     const { metricName, canaryMetricConfig, metricSetPair, canaryAnalysisResult } = this.props;
 
-    // TODO below is just a stub showing that the data is available
     return (
-      <div>
-        {/*<div>Name: {metricName}</div>*/}
-        {/*<div>{canaryAnalysisResult.classification}</div>*/}
-        {/*<div>{canaryAnalysisResult.classificationReason}</div>*/}
-        {/*<div>{JSON.stringify(canaryMetricConfig.query)}</div>*/}
-        {/*<div>Number of control data points: {metricSetPair.values.control.length}</div>*/}
-        {/*<div>Number of experiment data points: {metricSetPair.values.experiment.length}</div>*/}
+      <div className="metric-container">
+        <div className="metric-name">{metricName}</div>
+        <div className="metric-symbols">
+          {canaryMetricConfig.analysisConfigurations.canary.critical && (
+            <div className="metric-critical-symbol">
+              <FontAwesomeIcon className="exclamation" size="1x" color="black" icon={faExclamation} />
+            </div>
+          )}
+          <div
+            className={[
```

Comment:
Please use the classnames package that we have in this project for better readability.

https://github.com/JedWatson/classnames#usage

Reply:
Per our IRL conversation this could be written in one of the following 2 ways and be a bit more readable

```javascript
className={classnames('dot', {
    pass: canaryAnalysisResult.classification === 'Pass'
    'no-data':  canaryAnalysisResult.classification === 'Nodata'
    fail: canaryAnalysisResult.classification === 'Fail'
})}
```

```javascript
className={classnames('dot', {
    [canaryAnalysisResult.classification.toLower]: true
})}
```